skglm/datafits/multi_task.py

```diff
@@ -91,3 +91,6 @@ def full_grad_sparse(self, X_data, X_indptr, X_indices, Y, XW):
                     XjTXW[t] += X_data[i] * XW[X_indices[i], t]
             grad[j, :] = (XjTXW - self.XtY[j, :]) / n_samples
         return grad
+
+    def intercept_update_step(self, Y, XW):
```

Comment:
use mean to be consistent as @PABannier suggested

Reply:
@mathurinm, mean with axis argument is unfortunately not supported in Numba.
It throws an error at JIT compilation time

Fixed that by using sum over n_samples